entries/c/cointracking.info.json

```diff
@@ -4,6 +4,7 @@
     "tfa": [
       "totp"
     ],
+    "recovery": "https://cointracking.freshdesk.com/en/support/solutions/articles/29000021946",
```

Comment:
Link doesn't work for me :thinking: 
![Unauthorized location](https://user-images.githubusercontent.com/3535780/157986418-20831ed6-5096-449d-89eb-745f0a21c939.png)